config/database.yml

```diff
@@ -1,5 +1,7 @@
 development:
   adapter: postgresql
+  host: <%= ENV['PGHOST'] || 'localhost' %>
```

Comment:
Because of the way my postgres install is set up, having these fallbacks breaks it (homebrew uses your username for your postgres user).

Removing the fallback resolved the issue for me (done in #1575) - hopefully that will work for you here too.

Though if you set the variable `POSTGRESQL_USERNAME` rather than `PGUSER` it might work without needing to add these keys?

Reply:
Good point, actually with `PGUSER` already works without the keys. It was working from the test database already! Probably I needed when doing initial configuration for debugging, but don't need anymore!